src/components/WarningAnimateMessage/index.js

```diff
@@ -41,7 +41,7 @@ const WarningAnimatedMessage = ({isSHow, top = 100, left = 110}) => {
       useNativeDriver: true
     }).start(() => {
       setIsRunAnimated(false);
-      saveToCache(OPEN_FIRST_TIME, {isOpen: true});
+      StorageUtils.incognitoCreatePostFirstTime.set('true');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to line 31, the storage operation has been updated to use `StorageUtils.incognitoCreatePostFirstTime.set('true')` instead of `saveToCache(OPEN_FIRST_TIME, {isOpen: true})`. This change improves code readability and maintainability by using a more descriptive key and a simpler value.
```diff
- saveToCache(OPEN_FIRST_TIME, {isOpen: true});
+ StorageUtils.incognitoCreatePostFirstTime.set('true');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->